lib/multi.c

```diff
@@ -1140,6 +1171,24 @@ static CURLMcode Curl_multi_wait(struct Curl_multi *multi,
 
         extra_fds[i].revents = mask;
       }
+
+#ifdef ENABLE_UNBLOCK
+      if(multi->unblock_read_socket != CURL_SOCKET_BAD) {
+        if(ufds[curlfds + extra_nfds].revents & POLLIN) {
+          char buf[64];
```

Comment:
Reading the `64` here it made me wonder if A) should it be documented or not? and B) should this perhaps rather "drain" the socket ie read until `EWOULDBLOCK` to handle more unblocks than would fit in a single buffer? or C) maybe just leave it as-is and deal with it in a future should anyone ever actually get affected by it.

Reply:
I went with option B